ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/metrics/MetricsReporting.scala

```diff
@@ -35,25 +31,16 @@ final class MetricsManager(jmxDomain: String, enableJmxReporter: Boolean) extend
 
   if (enableJmxReporter) jmxReporter.start()
 
-  def timedFuture[T](timerName: String, f: => Future[T]): Future[T] = {
-    val timer = metrics.timer(timerName)
-    timedFuture(timer, f)
-  }
-
-  def timedFuture[T](timer: Timer, f: => Future[T]): Future[T] = {
-    val ctx = timer.time()
-    val res = f
-    res.onComplete(_ => ctx.stop())(DirectExecutionContext)
-    res
-  }
-
   override def close(): Unit = {
     slf4jReporter.report()
     if (enableJmxReporter) jmxReporter.close()
   }
 }
 
-object MetricsManager {
-  def apply(jmxDomain: String, enableJmxReporter: Boolean = true): MetricsManager =
-    new MetricsManager(jmxDomain, enableJmxReporter)
+object MetricsReporting {
+  def apply(
+      metrics: MetricRegistry,
+      jmxDomain: String,
+      enableJmxReporter: Boolean = true): MetricsReporting =
+    new MetricsReporting(metrics, jmxDomain, enableJmxReporter)
```

Comment:
Why bother with this? We can just call the constructor directly.

Reply:
Consider it gone.

Reply:
Addressed by 1227da2.